docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md

```diff
@@ -0,0 +1,736 @@
+# Migration Guide
+
+Upgrading your Strapi application to `v3.0.0-beta.x`.
+
+To upgrade a project to the `beta` version of Strapi follow the instructions below.
+
+## Updating global Strapi installation
+
+If you have installed Strapi globally. You will need to upgrade its version:
+
+```bash
+npm install -g strapi@beta
+# or
+yarn global add strapi@beta
+```
+
+## Updating dependencies
+
+First let's clean your project's dependencies and update its package.json file.
```

Comment:
First, let's clean your project's dependencies and update its package.json file.

Add a ","